locale.json

```diff
@@ -69,5 +90,53 @@
             "prevChapter": "Vorheriges Kapitel",
             "nextChapter": "Nächstes Kapitel"
         }
+    },
+    "ja": {
+        "langName": "Japanese",
```

Comment:
We should probalby have this in Japanese as well (unless you think that'd be weird). It'd make it consistent with the other languages.